java/src/main/kotlin/io/spine/validation/java/ValidationConstraintsCode.kt

```diff
@@ -126,7 +126,7 @@ internal class ValidationConstraintsCode private constructor(
         /**
          * Creates a new instance with the generated validation constraints code.
          */
-        fun generate(r: JavaValidationRenderer, v: MessageValidation): ValidationConstraintsCode {
+        fun generate(r: JavaValidationRenderer, v: CompiledMessage): ValidationConstraintsCode {
```

Comment:
As a general comment: when renaming something, please check the corresponding variable names, too.